app/Config/core.php

```diff
@@ -149,13 +149,13 @@
  *	`manager_index()` and `/manager/controller/index`
  *
  */
-	//Configure::write('Routing.prefixes', array('admin'));
+    //Configure::write('Routing.prefixes', array('admin'));
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Routing.prefixes', array('admin'));" but found "//Configure::write('Routing.prefixes', array('admin'));"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment